sns_ik_lib/matlab/TEST_acc_ik_solvers.m

```diff
@@ -0,0 +1,18 @@
+% This script sets up an runs a set of unit tests on the acceleration IK solvers.
```

Comment:
We'll need a license file in this folder, and an abbreviated license with a Rethink copyright at the top of each matlab file.